Client/Experiments/Experiments.swift

```diff
@@ -136,10 +136,14 @@ enum Experiments {
 
         let isPhone = UIDevice.current.userInterfaceIdiom == .phone
 
+        let prefs = UserDefaults(suiteName: AppInfo.sharedContainerIdentifier)!
```

Comment:
I am uncomfortable with force unwrapping in prod code (and that swiftlint rule is coming soon). Can we unwrap this?